### DISCUSSION TITLE
compute intensities Error

### DISCUSSION THREAD

**Discussion by MohammadReza0GH** [phoebe-project/phoebe2#868] [Questions]
hi in lc_geometry and Optimizer i recieve this error (i did set atm to blackbody):
 
in Optimizer :
single model exceeded number of allowed retries.  Check sampling distribution to ensure physical models exist.  Latest model error: Could not compute intensities with atm='ck2004'.  Try changing atm to a table that covers a sufficient range of values (or to 'blackbody' in which case ld_mode will need to be set to 'manual' and coefficients provided via ld_coeffs).  Enable 'warning' logger to see out-of-bounds arrays.

in lc_geometry:

Could not compute ldint with ldatm='ck2004'.  Try changing atm to a table that covers a sufficient range of values.  If necessary, set atm to 'blackbody' and/or ld_mode to 'manual' (in which case coefficients will need to be explicitly provided via ld_coeffs). Enable 'warning' logger to see out-of-bound arrays.

**Comment by kecnry** [comment]
These error messages are telling you that the input stellar parameters (temperature, logg, etc) are outside the atmosphere grids and therefore are failing and suggesting that you either check your stellar parameters to ensure they are physical, or you can instead use either a different model atmosphere or the blackbody approximation - whichever is most appropriate for your system and science case.

**Reply by MohammadReza0GH** [reply]
yes, tnx, but in our case, primary is a MS star and secondary is NS, atm for NS of course is blackbody but for primary it has to be ck2004 or phoenix for good approximation but same error appears.
Teff,1 = 10^6 K -> blackbody
Teff,2 = 4000 K -> ck2004 or phoenix -> error

**Reply by kecnry** [reply]
Can you enable the warning logger `phoebe.logger('warning')` and then call `run_compute()` as suggested by the error message? - this should tell you what elements are falling out-of-bounds.

**Reply by MohammadReza0GH** [reply]
WARNING Atmosphere parameters out of bounds: ldatm=ck2004, teff=[3950.], logg=[6.04098886], abun=[0.]

**Reply by aprsa** [reply]
logg=6.04 is too large for ck2004.

**Reply by MohammadReza0GH** [reply]
tnx... and one more question... how can i put mass1 and mass2 only in mcmc and see masses distribution?
